### PR TITLE
DBZ-5268 Removing clientSetName command in Redis to be able to use GCP Managed Redis

### PR DESCRIPTION
**Pull request by rahulkhanna2**
**GCP Managed Redis does not support client commands.** 

**Solution:** 
Removing clientSetName to be able to use GCP Managed Redis

### CODE REVIEW

**Comment by spicy-sauce** [comment]
I don't think that removing this is the right approach here as it's useful when using Redis OSS, Redis Enterprise, etc.
If for some reason this capability is not supported when using GCP, I'd just wrap it with try and catch block and swallow the exception.

**Comment by jpechane** [comment]
@spicy-sauce +1, just instead of plain swallowing please log the situation at the `WARN` level

**Comment by rahulkhanna2** [comment]
> @spicy-sauce +1, just instead of plain swallowing please log the situation at the `WARN` level

Thanks for the suggestion. I have logged the exception.

**Comment by jpechane** [comment]
@rahulkhanna2 Could you please build the PR locally so the code is properly formatted? Thanks

**Comment by rahulkhanna-fk** [comment]
> @rahulkhanna2 Could you please build the PR locally so the code is properly formatted? Thanks

I have done `mvn clean install ` with skipping tests on local and it did build successfully. Should work now. Can you rerun. 

Thanks

**Comment by spicy-sauce** [comment]
LGTM

**Comment by spicy-sauce** [comment]
> Welcome as a new contributor to Debezium, @rahulkhanna2. Reviewers, please add missing author name(s) and alias name(s) to the [COPYRIGHT.txt](https://github.com/debezium/debezium/blob/main/COPYRIGHT.txt) and [Aliases.txt](https://github.com/debezium/debezium/blob/main/jenkins-jobs/scripts/config/Aliases.txt) respectively.

@rahulkhanna-fk please follow this too.

**Comment by rahulkhanna2** [comment]
> > Welcome as a new contributor to Debezium, @rahulkhanna2. Reviewers, please add missing author name(s) and alias name(s) to the [COPYRIGHT.txt](https://github.com/debezium/debezium/blob/main/COPYRIGHT.txt) and [Aliases.txt](https://github.com/debezium/debezium/blob/main/jenkins-jobs/scripts/config/Aliases.txt) respectively.
> 
> @rahulkhanna-fk please follow this too.

Done. Thanks

File: debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisConnection.java [hunk 1]
```diff
@@ -55,7 +56,13 @@ else if (this.password != null) {
             client.ping();
         }
 
-        client.clientSetname(clientName);
+        try {
+            client.clientSetname(clientName);
+        }
+        catch (JedisDataException e) {
+            LOGGER.warn("Failed to set client name ", e);
```
[the remark this answers is not in the document]
Comment (rahulkhanna2) [reply debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisConnection.java]: Yes. Just quick typing I guess. Thanks

**Review by spicy-sauce — APPROVED** [review]

**Comment by jpechane** [comment]
@rahulkhanna-fk We are done, could you please squash the commits into a single one and I'll merge the PR and backport it to 1.9 too

**Comment by rahulkhanna2** [comment]
> @rahulkhanna-fk We are done, could you please squash the commits into a single one and I'll merge the PR and backport it to 1.9 too

Done

**Comment by jpechane** [comment]
@rahulkhanna2 Applied, thanks a lot!

**Pull request merged**